satpy/etc/enhancements/generic.yaml

```diff
@@ -359,8 +359,8 @@ enhancements:
         palettes:
           - dataset: ct_pal
             color_scale: 255
-            min_value: 0
-            max_value: 255
+            # NB: setting min_value and max_value that differ from the valid_range in the data
+            # will result in wrong colors.  It's safer to not set min_value and max_value at all.
```

Comment:
If we have a section about palettes in the documentation, I think it would be nice to add these lines to it

Reply:
perfect thanks! Merging